boards/nucleo-f103/Makefile.features

```diff
@@ -0,0 +1,8 @@
+FEATURES_PROVIDED += cpp
```

Comment:
Could you please use the same template as e.g. in `boards/stm32f4discovery/Makefile.features` for this file to keep all the Makefile.features consistent? Thx!